srtcore/core.cpp

```diff
@@ -213,6 +216,31 @@ struct SrtOptionAction
 }
 srt_options_action;
 
+#if HAVE_CXX11
+
+CUDTUnited* CUDT::uglobal()
+{
+    static CUDTUnited instance;
+    return &instance;
+}
```

Comment:
To reduce dereferencing, this would be a more convenient function.

```suggestion
CUDTUnited& CUDT::uglobal()
{
    static CUDTUnited instance;
    return instance;
}
```

Reply:
I don't think there is performance difference between pointer and reference. They are compiled to the same machine code.

If the singleton needs to be deleted as the previous review comments, then delete a reference looks odd.